src/tsconfig.ts

```diff
@@ -43,18 +44,21 @@ const recommended: Record<string, any> = {
   },
 }
 
-const build: Record<string, any> = {
+const build = (): Record<string, any> => ({
   extends:
     config.project === undefined
       ? '../tsconfig.json'
       : join('..', config.project),
   compilerOptions: {
+    target:
+      readTypescriptConfig().options.target === undefined
+        ? 'es2022'
+        : undefined,
```

Comment:
```suggestion
        target: readTypeScriptConfig().options.target ?? 'es2022'
```

Reply:
What we're doing here is subtly different from that:
- if the target is undefined in the tsconfig.json, then we set es2022
- if the target is defined, then we don't set it _at all_, so that it inherits from the base tsconfig.

With your suggestion we are instead setting the target in build.json to be the same as in the base tsconfig -- we _could_ do that, but then we'd need to pull in a bit more logic from TS to stringify the value again (the parsed value is actually a `ScriptTarget`, which TS defines as a numeric enum).

Reply:
Oic, that makes sense. Thanks for the clarification.